src/arfsdao.ts

```diff
@@ -73,9 +72,110 @@ export interface ArFSCreatePrivateDriveResult extends ArFSCreateDriveResult {
 	driveKey: DriveKey;
 }
 
-export class ArFSDAO {
+/**
+ * contains all queries that doesn't need for a wallet
+ */
+export class ArFSReadOnlyDAO {
```

Comment:
I think it's a little tricky to break up ArFS and DAO in the name like this. I'd follow the naming pattern you used above with ArDriveReadOnly, i.e. ArFSDAOReadOnly

Reply:
Renamed at 5feaea5